### PR TITLE
feat(camunda-cloud): allow replace Message Intermediate Throw and Message End events 

### PR DESCRIPTION
**Pull request by smbea**
Related to https://github.com/camunda/camunda-modeler/issues/2420

Also adds tests related to replacing tasks with ManualTask, missing in https://github.com/camunda/camunda-bpmn-js/pull/49

### CODE REVIEW

File: test/camunda-cloud/features/popup-menu/ReplaceMenuProviderSpec.js [hunk 1]
```diff
@@ -66,12 +66,14 @@ describe('camunda-cloud/features - PopupMenu', function() {
 
       const startEventEntry = queryEntry(popupMenu, 'replace-with-none-start'),
             intermediateEventEntry = queryEntry(popupMenu, 'replace-with-none-intermediate-throw'),
-            errorEndEventEntry = queryEntry(popupMenu, 'replace-with-error-end');
+            errorEndEventEntry = queryEntry(popupMenu, 'replace-with-error-end'),
+            messageEndEventEntry = queryEntry(popupMenu, 'replace-with-message-end');
```
Comment (MaxTru) [inline test/camunda-cloud/features/popup-menu/ReplaceMenuProviderSpec.js:70]: Let's also add a new test case to check whether we can replace from a messageThrow and a messageEnd event to the repsective other events. E.g.;
```
    it('should contain options for MessageThrowEvent', ...
```

and

```
    it('should contain options for MessageEndEvent', ...
```

**Review by MaxTru — COMMENTED** [review]
Looks good! Only one small comment, afterwards: ready to merge

**Comment by MaxTru** [comment]
Cool! 

Minor, but still relevant: we should limit git commit message summary line to max. of 72 characters to avoid linebreaks like this:
Screenshot_20210916_161917

**Comment by smbea** [comment]
> Cool!
> 
> Minor, but still relevant: we should limit git commit message summary line to max. of 72 characters to avoid linebreaks like this:
> Screenshot_20210916_161917

Got it! Shortened the commit message

**Review by MaxTru — APPROVED** [review]

**Pull request merged**